internal/pkg/fixtures/configs/shared-codes.json

```diff
@@ -24,7 +24,7 @@
       "isDisabled": false,
       "configuration": {
         "code_content": "num1 = {{num1}}\nnum2 = {{num2}}\nsum = num1 + num2",
-        "variables_id": "..."
+        "variables_id": "%%TEST_SHARED_CODE_VARIABLES_ID%%"
```

Comment:
Mame tu testovaci config vo `fixtures/configs`, ktory ma mat v sebe `variables_id`, tj. ID inej konfiguracie.
To doteraz nebolo mozne.

Pouzijeme tu premennu `TEST_SHARED_CODE_VARIABLES_ID`, kedze nevieme v akej branchi sa tento config realne pouzije.